unit_testing/test_sct_compute_mtsat.py

```diff
@@ -9,20 +9,28 @@
 
 import sct_compute_mtsat
 
+out_mstat = "out_mtsat.nii.gz"
+out_t1map = "out_t1map.nii.gz"
 
 INPUT_PARAMS = [
     ['-mt', sct_test_path('mt', 'mt1.nii.gz'),
      '-pd', sct_test_path('mt', 'mt0.nii.gz'),
-     '-t1', sct_test_path('mt', 't1w.nii.gz')],
+     '-t1', sct_test_path('mt', 't1w.nii.gz'),
+     '-omtsat', out_mstat,
+     '-ot1map', out_t1map],
     ['-mt', sct_test_path('mt', 'mt1.nii.gz'),
      '-pd', sct_test_path('mt', 'mt0.nii.gz'),
      '-t1', sct_test_path('mt', 't1w.nii.gz'),
+     '-omtsat', out_mstat,
+     '-ot1map', out_t1map,
      '-trmt', '51', '-trpd', '52', '-trt1', '10', '-famt', '4', '-fapd', '5', '-fat1', '14'],
     ]
 
 
 @pytest.mark.parametrize('input_params', INPUT_PARAMS)
 def test_with_json_sidecar(input_params):
     sct_compute_mtsat.main(input_params)
-    # Check if output file exists
-    assert os.path.isfile(sct_test_path('mt', 'mtsat.nii.gz'))
+    # Check if output files exist
+    for f in [out_mstat, out_t1map]:
```

Comment:
Nitpick: Since we're already using `pytest.mark.parameterize`, could these paths be accessed using `input_params['-omtsat']` and `input_params['-ot1map']`?

That way, we wouldn't need to create and access these variables:
```
out_mstat = "out_mtsat.nii.gz"
out_t1map = "out_t1map.nii.gz"
```

Reply:
I agree that it would be nice to parametrize, but `input_params` is a list at the moment. I guess it's possible to make it a dict and transform it prior to passing to the script, but I don't think its worth it.

What I like about explicitly defining the output files is that it makes the code more readable as opposed to not defining them and relying on the default values (don't need to guess what those are).

Reply:
Sorry, you're absolutely right. I misread! It functions so much like a dict that I read it as if it were one.